packages/react-core/src/layouts/Level/Level.tsx

```diff
@@ -18,10 +17,7 @@ export const Level: React.FunctionComponent<LevelProps> = ({
   children = null,
   ...props
 }: LevelProps) => (
-  <div
-    {...props}
-    className={css(styles.level, gutter && getGutterModifier(styles, gutter, styles.modifiers.gutter), className)}
-  >
+  <div {...props} className={css(styles.level, gutter && styles.modifiers.gutter, className)}>
```

Comment:
previously it was returning `gutter-${size}` where size is one of sm, md, lg

Reply:
Gutters haven't had sizes for over a year now. They're either on or off.